test/algorithms/overlay/overlay_cases.hpp

```diff
@@ -1231,6 +1231,59 @@ static std::string issue_1342_b[2] =
     "POLYGON((2.000000861538488 2.944711629068049e-16,2.000000861538488 2.0000008,23.52786548447633 2.0000008,23.52786548447633 2.944711629068049e-16,2.000000861538488 2.944711629068049e-16))"
 };
 
+static std::string issue_1345_a[2] =
+{
+    // Needs check for opposite in handle_imperfect_touch
+    R""""(
```

Comment:
@vissarion this is (AFAIK) the first case written in this new multi-line format.
It doesn't have all those quotes and reflects the reported case.
Like it?

Reply:
Looks good, thanks.